types/templates/coloredscripthash/index.d.ts

```diff
@@ -0,0 +1,2 @@
+import * as output from './output';
+export { output };
```

Comment:
Isn't input also necessary?

Reply:
"ts_src/templates/coloredscripthash/input.ts" is not needed, so I'll remove this file.
(f9b28ab8c643f4f92f4b740d9b91d295dc3ad707)